src/agent/onefuzz-supervisor/src/config.rs

```diff
@@ -102,6 +108,10 @@ impl StaticConfig {
         let client_id = Uuid::parse_str(&std::env::var("ONEFUZZ_CLIENT_ID")?)?;
         let client_secret = std::env::var("ONEFUZZ_CLIENT_SECRET")?;
         let tenant = std::env::var("ONEFUZZ_TENANT")?;
+        let multi_tenant_domain = match std::env::var("ONEFUZZ_MULTI_TENANT_DOMAIN") {
```

Comment:
This can be simplified to:
```suggestion
        let multi_tenant_domain = std::env::var("ONEFUZZ_MULTI_TENANT_DOMAIN").ok();
```

[Result.ok](https://doc.rust-lang.org/std/result/enum.Result.html#method.ok) converts a Result<T> into an Option<T>.